editions/tw5.com/tiddlers/wikitext/Widgets in WikiText.tid

```diff
@@ -20,6 +20,7 @@ Note that widgets inherit all the features of [[HTML in WikiText]]:
 ** Macro invocations (eg `attr=<<myMacro>>`)
 ** Transclusions (eg, `attr={{MyTiddler!!field}}`)
 ** Filtered transclusions (eg, `attr={{{ [filter[op]] }}}`)
+** <<.from-version "5.3.0">> see also [[Substituted Attribute Values]]
```

Comment:
Recommend dropping the words "see also" for consistency.

Reply:
If that is somehow consistent, sure do so, however it looks more readable to me, since it is under a list with the title;

**Widget attributes can be specified as:**

As in another recent documentation change, I do not know where to go now, in the GitHuib process to accept or reject your comments, if I wanted to apply the alternative etc...

Reply:
Instructions for updating a PR using the Github web UI: https://ffeathers.wordpress.com/2020/01/12/how-to-update-your-pr-on-github/#:~:text=Open%20a%20browser%20window.,editing%20interface%20that%20opens%20up.

Reply:
As the creator of the PR you should be able to "Edit" your content from the web-UI here.

![image](https://github.com/Jermolene/TiddlyWiki5/assets/374655/b301b5aa-66f0-4a66-842d-8d58059cf07a)